apisix/admin/init.lua

```diff
@@ -258,17 +258,21 @@ local function sync_local_conf_to_etcd(reset)
     local local_conf = core.config.local_conf()
 
     local plugins = {}
-    for _, name in ipairs(local_conf.plugins) do
-        core.table.insert(plugins, {
-            name = name,
-        })
+    if local_conf.plugins then
```

Comment:
In what situation people will use an empty plugin list?

Reply:
In most cases the list of http plugins is not empty, but the list of stream plugins is empty.

Reply:
In the production environment we turned off all unused plugins.